images/svc-base/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM registry.scality.com/federation/nodesvc-base:7.10.6.0
```

Comment:
Would it be possible to create a `nodesvc-base:7.10` that is updated with each 7.10 release and we can use that? I don't think we can/should rely on either passing in a federation version or expecting the federation version to already exist. It works in leef because its inside federation so the VERSION value is always going to match.

Reply:
Floating tags generally are to be avoided for traceability and reproducibility reasons.

Reply:
Yea yea I know... Just seemed slightly easier then relying on a federation version and constantly updating a full version tag without automation... 😦

Reply:
There definitely needs to be some automation.

Reply:
@Emylou-s have this subject on the view, where maybe (still a work/design in progress) svc-base will be removed from federation. From this point it will have it's own tag which will move at it's own pace.

(Emylou feel free to correct me if mistaken)

Reply:
Indeed, that's a another why we should be doing it.
I'll add it to the pile of good arguments

https://scality.atlassian.net/browse/S3C-6432